### PR TITLE
Modified Task_DEV_3

### PR DESCRIPTION
**Pull request by PolinaYeroshevich**
Finding a prime number in an array using methods.

### CODE REVIEW

File: Task_DEV_3/Task_DEV_3/Program.cs [hunk 1]
```diff
@@ -4,39 +4,49 @@ namespace Task_DEV_3
...
     class Program
     {
+        static void InputArrayElement(int[] inputNumbers, int arrayLength)
+        {
+            for (int i = 0; i < arrayLength; i++)
+            {
+                Console.WriteLine($"Enter an array element № {i}");
+                inputNumbers[i] = Convert.ToInt32(Console.ReadLine());
+            }
+        }
```
Comment (VadimBelkin) [inline Task_DEV_3/Task_DEV_3/Program.cs:16]: Need to add blank line between methods.

File: Task_DEV_3/Task_DEV_3/Program.cs [hunk 2]
```diff
@@ -4,39 +4,49 @@ namespace Task_DEV_3
...
+                if (isPrimeNumber)
+                {
+                    Console.WriteLine($"{inputNumbers[i]} is prime number");
+                }
+                else
+                {
+                    Console.WriteLine($"{inputNumbers[i]} is not prime number");
+                }
+            }
+        }
```
Comment (VadimBelkin) [inline Task_DEV_3/Task_DEV_3/Program.cs:30]: Need to add blank line between methods.

File: Task_DEV_3/Task_DEV_3/Program.cs [hunk 3]
```diff
@@ -4,39 +4,49 @@ namespace Task_DEV_3
 {
     class Program
     {
+        static void InputArrayElement(int[] inputNumbers, int arrayLength)
```
Comment (VadimBelkin) [inline Task_DEV_3/Task_DEV_3/Program.cs]: Please do smth like
static int[] CreateNambersArray(int arrayLength)

File: Task_DEV_3/Task_DEV_3/Program.cs [hunk 4]
```diff
@@ -4,39 +4,49 @@ namespace Task_DEV_3
...
+        {
+            for (int i = 0; i < arrayLength; i++)
+            {
+                Console.WriteLine($"Enter an array element № {i}");
+                inputNumbers[i] = Convert.ToInt32(Console.ReadLine());
+            }
+        }
+        static void CheckingNumbersForPrime(int[] inputNumbers, int arrayLength)
```
Comment (VadimBelkin) [inline Task_DEV_3/Task_DEV_3/Program.cs]: Would be better to create method which checks one number and return `true `or `false`.

Comment (VadimBelkin) [reply Task_DEV_3/Task_DEV_3/Program.cs]: The first word in method name should be verb.

File: Task_DEV_3/Task_DEV_3/Program.cs [hunk 5]
```diff
@@ -4,39 +4,49 @@ namespace Task_DEV_3
...
+                {
+                    if (inputNumbers[i] % j == 0)
+                    {
+                        isPrimeNumber = false;
+                        break;
+                    }
+                }
+                if (isPrimeNumber)
+                {
+                    Console.WriteLine($"{inputNumbers[i]} is prime number");
```
Comment (VadimBelkin) [inline Task_DEV_3/Task_DEV_3/Program.cs]: Single responsibility violation. If method checks smth it doesn't need to print smth to console.

File: Task_DEV_3/Task_DEV_3/Program.cs [hunk 6]
```diff
@@ -4,39 +4,49 @@ namespace Task_DEV_3
...
+                }
+                else
+                {
+                    Console.WriteLine($"{inputNumbers[i]} is not prime number");
```
Comment (VadimBelkin) [inline Task_DEV_3/Task_DEV_3/Program.cs]: Single responsibility violation. If method checks smth it doesn't need to print smth to console.

File: Task_DEV_3/Task_DEV_3/Program.cs [hunk 7]
```diff
@@ -4,39 +4,49 @@ namespace Task_DEV_3
...
-                    if (isPrimeNumber)
-                    {
-                        Console.WriteLine($"{inputNumbers[i]} is prime number");
-                    }
-                    else
-                    {
-                        Console.WriteLine($"{inputNumbers[i]} is not prime number");
-                    }
-                }
+
```
Comment (VadimBelkin) [inline Task_DEV_3/Task_DEV_3/Program.cs]: Unnecessary blank line.

File: Task_DEV_3/Task_DEV_3/Program.cs [hunk 8]
```diff
@@ -4,39 +4,49 @@ namespace Task_DEV_3
...
+                InputArrayElement(inputNumbers, arrayLength);
+
```
Comment (VadimBelkin) [inline Task_DEV_3/Task_DEV_3/Program.cs]: Unnecessary blank line.

**Review by VadimBelkin — CHANGES_REQUESTED** [review]

File: Task_DEV_3/Task_DEV_3/Program.cs [hunk 9]
```diff
@@ -4,30 +4,41 @@ namespace Task_DEV_3
...
+            int[] inputNumbers = new int[arrayLength];
+            for (int i = 0; i < arrayLength; i++)
+            {
+                Console.WriteLine($"Enter an array element № {i}");
+                inputNumbers[i] = Convert.ToInt32(Console.ReadLine());
+            }
+            return inputNumbers;
+        }
+
+        static bool CheckNumbersForPrime(int number)
```
Comment (VadimBelkin) [inline Task_DEV_3/Task_DEV_3/Program.cs]: Numbers or number?

File: Task_DEV_3/Task_DEV_3/Program.cs [hunk 10]
```diff
@@ -4,30 +4,41 @@ namespace Task_DEV_3
...
-                    isPrimeNumber = true;
-                    for (int j = 2; j < inputNumbers[i]; j++)
-                    {
-                        if (inputNumbers[i] % j == 0)
-                        {
-                            isPrimeNumber = false;
-                            break;
-                        }
-                    }
+                    bool isPrimeNumber = CheckNumbersForPrime(inputNumbers[i]);
```
Comment (VadimBelkin) [inline Task_DEV_3/Task_DEV_3/Program.cs]: You don't even need to create a separate variable `isPrimeNumber`. It is possible to do smth like:
`if(CheckNumbersForPrime(inputNumbers[i]))`

**Review by VadimBelkin — CHANGES_REQUESTED** [review]

File: Task_DEV_3/Task_DEV_3/Program.cs (hunk 9 above)
Comment (PolinaYeroshevich) [reply Task_DEV_3/Task_DEV_3/Program.cs]: Done.

File: Task_DEV_3/Task_DEV_3/Program.cs (hunk 10 above)
Comment (PolinaYeroshevich) [reply Task_DEV_3/Task_DEV_3/Program.cs]: Done.

**Review by VadimBelkin — APPROVED** [review]
Looks good for me.

**Review by MariaEroshevich — APPROVED** [review]

**Pull request merged**